bundles/mapping/shadow-plugin-3d/plugin/ShadowingPluginClass.js

```diff
@@ -46,15 +45,6 @@ class ShadowingPlugin extends BasicMapModulePlugin {
         this._createUI(mapInMobileMode);
     }
 
-    _createEventHandlers () {
-        return {
-            TimeChangedEvent: function (event) {
```

Comment:
This would work better with RPC etc programmatic changes to time done outside this UI.